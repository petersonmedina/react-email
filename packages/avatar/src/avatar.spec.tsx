import { Avatar } from './index';
import { render } from '@react-email/render';

describe('render', () => {
  beforeEach(() => {
    jest.restoreAllMocks();
    jest.resetModules();
  });

  it('renders the <Avatar> component', () => {
    const actualOutput = render(<Avatar src="cat.jpg" name=" My Cat" />);
    expect(actualOutput).toMatchSnapshot();
  });
});
