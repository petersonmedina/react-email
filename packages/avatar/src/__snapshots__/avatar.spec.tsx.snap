// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders the <Avatar> component 1`] = `"<!DOCTYPE htmlPUBLIC \\"-//W3C//DTD XHTML 1.0 Transitional//EN\\" \\"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\\"><span id=\\"avatarContainer\\" style=\\"display:grid;align-items:center;justify-content:center;border-radius:100%;overflow:hidden;background-color:#eaeaea;box-shadow:0px 0px 10px rgba(0, 0, 0, 0.12);width:60px;height:60px\\"><label style=\\"display:inline-flex;align-items:center;justify-content:center;width:60px;height:60px;font-size:24px\\">undefinedC</label><span style=\\"width:60px;height:60px;background-size:100%;background-image:url(cat.jpg);background-repeat:no-repeat;border-radius:100%;margin-top:-100%\\"></span></span>"`;
